scripts/export_comments.py

```diff
@@ -0,0 +1,103 @@
+import sys
+sys.path.append('..')
+import os
+import time
+
+from openpyxl import Workbook
+from sqlalchemy import create_engine
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.types import Boolean
+from sqlalchemy.dialects.postgresql import JSONB, UUID, TIMESTAMP
+
+import settings
+
+try:
+    db = create_engine(settings.DB_URI)
+    Session = sessionmaker(db)
+    session = Session()
+    base = declarative_base()
+except SQLAlchemyError as e:
+    print(e)
+    raise
+
+
+class SurveyResponse(base):
+    __tablename__ = 'responses'
+    tx_id = Column("tx_id",
+                   UUID,
+                   primary_key=True)
+
+    ts = Column("ts",
+                TIMESTAMP(timezone=True))
+
+    invalid = Column("invalid",
+                     Boolean,
+                     default=False)
+
+    data = Column("data", JSONB)
+
+    def __init__(self, tx_id, invalid, data):
+        self.tx_id = tx_id
+        self.invalid = invalid
+        self.data = data
+
+    def __repr__(self):
+        return '<SurveyResponse {}>'.format(self.tx_id)
+
+    def to_dict(self):
+        return {c.name: getattr(self, c.name) for c in self.__table__.columns}
+
+
+def create_comments_book(survey_id, comments):
+    print("Generating Excel file")
+    wb = Workbook()
+    row = 2
+    ws = wb.active
+    ws.cell(1, 1, "Survey ID: {}".format(survey_id))
+    ws.cell(1, 2, "Comments found: {}".format(str(len(comments))))
+    for comment in comments:
+        row += 1
+        ws.cell(row, 1, comment.data['data']['146'])
```

Comment:
Could `146` be pulled out into a variable with a meaningful name? It has no context here.

Reply:
Agreed and done